app/scripts/modules/core/pipeline/config/triggers/git/gitTrigger.html

```diff
@@ -0,0 +1,46 @@
+<ng-form name="cronForm">
+  <div class="form-group">
+    <label class="col-md-3 sm-label-right">
+      Repository Type
+      <help-field key="pipeline.config.git.repotype"></help-field>
+    </label>
+    <div class="col-md-6">
+      <select
+        class="form-control input-sm"
+        ng-options="repoType for repoType in ['stash', 'github']"
+        ng-model="vm.trigger.source">
+        <option value="">Select Repo Type</option>
+      </select>
+    </div>
+  </div>
+  <div class="form-group">
+    <label class="col-md-3 sm-label-right">
+      Project
+      <help-field key="pipeline.config.git.project"></help-field>
+    </label>
+
+    <div class="col-md-9">
+      <input type="text" class="form-control input-sm"
+             ng-model="vm.trigger.project"
+             name="project"
+             required
+             placeholder="Enter your repository project name, i.e. spinnaker for github.com/spinnaker/echo"
```

Comment:
yes, was trying to keep consistent with the project attributes screen. Makes sense to use the terminology for each scm type in the UI even if we call it something else in the backend